### PR TITLE
[NUI] Add sound feedback for Control

### PR DESCRIPTION
**Pull request by zhouleonlei**
### Description of Change ###
Add sound feedback for Control

### CODE REVIEW

**Review by elishateng — APPROVED** [review]
LGTM

**Review by AdunFang — APPROVED** [review]
It is OK.

**Review by AchoWang — APPROVED** [review]
Looks good to me.

**Review by huayongxu — APPROVED** [review]
looks good to me.

File: src/Tizen.NUI.Components/Controls/Control.cs [hunk 1]
```diff
@@ -42,6 +44,8 @@ public class Control : VisualView
 
         private bool onThemeChangedEventOverrideChecker;
```
Comment (dongsug-song) [inline src/Tizen.NUI.Components/Controls/Control.cs:46]: Thank you for your work. 
But could you resolve the warning shown below? 
I know this is not your modification, but basically we are going without build warning.
=>
src/Tizen.NUI.Components/Controls/Control.cs#L38
Use nameof in place of string literal 'Command'

**Review by dongsug-song — COMMENTED** [review]
could you add some sample app or TC to verify this?
example) https://github.com/Samsung/TizenFX/commit/3f6d076a3f28d3cd15870a513bf5c111cbe5d5b4#diff-5906bf6dceb4877dbdb9691459e595d4c20d777187ef6a73de4a506c87f8048aR592

thank you.

File: src/Tizen.NUI.Components/Controls/Control.cs (hunk 1 above)
Comment (zhouleonlei) [reply src/Tizen.NUI.Components/Controls/Control.cs:46]: > Thank you for your work.
> But could you resolve the warning shown below?
> I know this is not your modification, but basically we are going without build warning.
> =>
> src/Tizen.NUI.Components/Controls/Control.cs#L38
> Use nameof in place of string literal 'Command'

Done

**Review by zhouleonlei — COMMENTED** [review]
> could you add some sample app or TC to verify this?
> example) [3f6d076#diff-5906bf6dceb4877dbdb9691459e595d4c20d777187ef6a73de4a506c87f8048aR592](https://github.com/Samsung/TizenFX/commit/3f6d076a3f28d3cd15870a513bf5c111cbe5d5b4#diff-5906bf6dceb4877dbdb9691459e595d4c20d777187ef6a73de4a506c87f8048aR592)
> 
> thank you.

Added sound feedback in Switch sample

**Review by zg2nets — APPROVED** [review]
LGTM

**Pull request merged**